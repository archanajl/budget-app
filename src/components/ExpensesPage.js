import React from 'react';
import ExpensesList from './ExpensesList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';


const ExpensesPage = () => (
    <div>
        <ExpenseListFilters/> 
        <ExpensesSummary />
        <ExpensesList/>
    </div>
)

export default ExpensesPage;