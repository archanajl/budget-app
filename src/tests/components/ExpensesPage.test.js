import React from 'react';
import {shallow} from 'enzyme'
import ExpensesPage from '../../components/ExpensesPage';

test('should render the expensesPage',()=>{
    const wrapper = shallow(<ExpensesPage />)
    expect(wrapper).toMatchSnapshot();
})