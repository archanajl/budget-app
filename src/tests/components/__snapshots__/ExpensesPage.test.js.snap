// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the expensesPage 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpensesSummary) />
  <Connect(ExpensesList) />
</div>
`;
